### PR TITLE
Match a couple functions related to text color

### PR DESCRIPTION
**Pull request by Tmcg2**
Also fix some set_text_color calls that weren't using the TEXT_* defines

Signed-off-by: Taggerung <tyler.taggerung@gmail.com>

### CODE REVIEW

File: src/code_80091750.c [hunk 1]
```diff
@@ -527,41 +528,28 @@ void func_800921B4(void) {
...
+void set_text_color(s32);
+extern s32 func_800B4520();
 
-void func_800921C0(s32 arg0, s32 arg1, s32 arg2) {
-    if (arg0 == arg1) {
+void func_800921C0(s32 test, s32 target, s32 alternative_color) {
+    if (test == target) {
         if (func_800B4520() != 0) {
             set_text_color(gGlobalTimer % 3);
             return;
```
Comment (MegaMech) [inline src/code_80091750.c:538]: When you see this pattern try:
```
        if (func_800B4520() != 0) {
            set_text_color(gGlobalTimer % 3);
            -return;
            else { 
            }
```

File: src/code_80091750.c [hunk 2]
```diff
@@ -527,41 +528,31 @@ void func_800921B4(void) {
...
-    if (arg0 == arg1) {
+void func_800921C0(s32 test, s32 target, s32 alternative_color) {
+    if (test == target) {
         if (func_800B4520() != 0) {
             set_text_color(gGlobalTimer % 3);
-            return;
         }
-        set_text_color(5);
-        return;
+        else{
```
Comment (MegaMech) [inline src/code_80091750.c:539]: else goes on the same line as the if closing bracket
```
if () {

} else {

}

```

**Review by MegaMech — CHANGES_REQUESTED** [review]

**Comment by MegaMech** [comment]
This is the style guide for most n64 decomp projects.

https://docs.google.com/document/d/1T_6dUj32QOscrMOMDsr7bY0esohswEAstKCO4gFn6t0/edit

File: src/code_80091750.c (hunk 1 above)
Comment (Tmcg2) [reply src/code_80091750.c]: Done

File: src/code_80091750.c (hunk 2 above)
Comment (Tmcg2) [reply src/code_80091750.c]: Done

**Review by MegaMech — APPROVED** [review]

**Pull request merged**